its/ruling/src/integrationTest/resources/sources/kubernetes/helm/corner-cases/templates/multiline-templates.yaml

```diff
@@ -0,0 +1,26 @@
+{{- /*
+  Line 1
+  Line 2
+*/ -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+  annotations:
+    {{-
+      if .Values.annotations
+    }} {{- range $key, $value := .Values.annotations }}
+    {{ $key }}: {{ $value | quote }} {{- end }}{{- end }}
+    cfg: |-
+      {{- if or
+        (eq "abc" .Values.foo)
+        (eq "def" .Values.bar)
+      }}
+      if
+      {{- else }}
+      else
+      {{- end }}
+spec:
```

Comment:
Please adapt this example to raise some issue to see that location shifting is working.